server/sonar-main/src/main/java/org/sonar/application/command/CommandFactoryImpl.java

```diff
@@ -67,27 +67,77 @@ public CommandFactoryImpl(Props props, File tempDir, System2 system2) {
   }
 
   @Override
-  public EsCommand createEsCommand() {
-    EsFileSystem esFileSystem = new EsFileSystem(props);
-    if (!esFileSystem.getExecutable().exists()) {
+  public AbstractCommand<?> createEsCommand() {
+    // FIXME reenable
+    //if (IS_OS_WINDOWS) {
+      return createEsCommandForWindows();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)